packages/nextjs/scaffold.config.ts

```diff
@@ -1,15 +1,17 @@
-import * as chains from "@starknet-react/chains";
+import { Chain } from "@starknet-react/chains";
+import { supportedChains as chains } from "./supportedChains";
 
 export type ScaffoldConfig = {
-  targetNetworks: readonly chains.Chain[];
+  targetNetworks: readonly Chain[];
   pollingInterval: number;
   onlyLocalBurnerWallet: boolean;
   rpcProviderUrl: string;
   walletAutoConnect: boolean;
+  autoConnectTTL: number;
 };
 
 const scaffoldConfig = {
-  targetNetworks: [chains.sepolia],
```

Comment:
migration is ok, but for our project we will use sepolia by default

Reply:
Done, changed default target network to sepolia